Rytis_George_Baltakys/fun_with_functions.py

```diff
@@ -0,0 +1,21 @@
+def odd_even():
+	for x in xrange(1,12):
+		oddeven = ''
+		if x % 2 == 0:
+			oddeven = 'even'
```

Comment:
You could choose to initialize it as odd and only change it if it goes into the first statement :)